docs/sources/features/datasources/prometheus.md

```diff
@@ -75,6 +75,9 @@ Name | Description
 
 For details of *metric names*, *label names* and *label values* are please refer to the [Prometheus documentation](http://prometheus.io/docs/concepts/data_model/#metric-names-and-labels).
 
+
+It is possible to use some global template variables in Prometheus query template variables; `$__interval`, `$__interval_ms`, `$__range` and `$__range_ms`, where `$__range` is the dashboard's current time range and `$__range_ms` is the current range in milliseconds.
```

Comment:
It's instead of It is? Think it would be nice to include the example used in test with a short description to underline the use case for these - or what do you think?

Reply:
Sounds reasonable! I'll figure out a good use case!

Reply:
Let me know if you want me to assist you with this

Reply:
I am probably going to need some help. I am not proficient enough with Prometheus to formulate something concise and understandable